Plugins/Sycl/include/Acts/Plugins/Sycl/Seeding/detail/Types.h

```diff
@@ -0,0 +1,65 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+#include <cstddef>
+
+#include <bits/stdint-uintn.h>
+
+namespace Acts::Sycl::detail {
+struct deviceSpacePoint {
```

Comment:
We will need input from the Acts developers, but I guess they took the convention from ATLAS of starting every class/struct name with a capital letter. (In ATLAS offline code we use this convention to more easily differentiate between "simple" functions and more complicated constructs like functors.)

Reply:
I would refer you to the [coding guidelines](https://acts.readthedocs.io/en/latest/codeguide.html), specifically the naming section.

Btw. I am not sure the `#include <bits/stdint-uintn.h>` is a standard include. Could you just use `#include <cstdint>` instead?

Reply:
I agree. I missed that include...

Reply:
Thanks

Reply:
The code still didn't change the capitalisation of these classes. :confused: The discussion should be kept open until the issue is resolved...